velox/exec/Spill.h

```diff
@@ -434,6 +434,41 @@ class SpillState {
 /// Generate partition id set from given spill partition set.
 SpillPartitionIdSet toSpillPartitionIdSet(
     const SpillPartitionSet& partitionSet);
+
+/// Testing utility for spills. This utility is only used in tests.
+class TestSpillUtils {
+ public:
+  /// Scoped spill percentage utility that allows user to set the chance of
+  /// spill. 100% means spill will always be triggered.
+  class TestScopedSpillPct {
+   public:
+    explicit TestScopedSpillPct(int32_t spillPct) {
+      prevSpillPct_ = testingSpillPct();
+      testingSetSpillPct(spillPct);
+    }
+
+    ~TestScopedSpillPct() {
+      testingSetSpillPct(prevSpillPct_);
+    }
+
+   private:
+    int32_t prevSpillPct_{0};
+  };
+
+  static int32_t testingGetSpillPct() {
```

Comment:
Move the following two utilities out of TestScopedSpillPct? Thanks!

Reply:
They are currently not in TestScopedSpillPct